App.js

```diff
@@ -0,0 +1,506 @@
+import React from 'react';
+import {useState, useEffect} from 'react';
+import type {Node} from 'react';
```

Comment:
Are you using flow for static typing, If not then I believe use .ts extension.
As Vscode linting error: `'import type' declarations can only be used in TypeScript files.`

Reply:
That is ok, that’s Vs code related setting, if you want you can safely rename all the js to ts , react native treat ts and js the same